moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -490,33 +490,44 @@ class RobotState
   /** \name Getting and setting joint positions, velocities, accelerations and effort
    *  @{
    */
+
+  /** \brief Sets position for 1 joint only. Check setVariablePositions() to set position values to multiple joints. */
```

Comment:
Please change all comments to this:
```suggestion
  ///@{
  /** \brief Set position(s) for a single, potentially multi-dof joint. See setVariablePositions() to handle multiple joints. */
```
Using [doxygen grouping](https://stackoverflow.com/questions/37284399/doxygen-single-comment-block-for-multiple-functions), we can also avoid duplication of the doc.

Reply:
Sure. Thank you for your feedback.
Also, the setJointPositions/Velocities/Efforts are already grouped in one. Do you want me further break them?
![doxygroup](https://user-images.githubusercontent.com/21956575/75655689-64f22d80-5c30-11ea-9d5b-b8bb70be43e3.png)

Reply:
Actually, I didn't notice that there is a general comment for the group already.
In this case, my suggestion is to slightly extend this general comment only.

Reply:
Made the changes.

Reply:
Please only modify the single grouping comment at the top. There is no need to repeat the same comment over and over.

Reply:
Is this better?

![Screenshot from 2020-03-02 03-46-11](https://user-images.githubusercontent.com/21956575/75659792-8b1bcb80-5c38-11ea-9156-0e2788dc907f.png)

Reply:
The screenshot looks good. However, this isn't yet committed, is it?

Reply:
Just committed this change now.